packages/wallet-service/package.json

```diff
@@ -36,22 +33,21 @@
     "serverless-mysql": "^1.5.4",
     "source-map-support": "^0.5.19",
     "tiny-secp256k1": "^2.2.1",
-    "uuid": "^8.3.0",
-    "winston": "^3.7.2"
+    "uuid": "^8.3.0"
   },
   "devDependencies": {
     "@types/aws-lambda": "^8.10.95",
     "@types/jest": "^27.0.24",
     "@types/node": "^18.0.4",
-    "@typescript-eslint/eslint-plugin": "^6.7.4",
-    "@typescript-eslint/parser": "^3.3.0",
+    "@typescript-eslint/eslint-plugin": "^7.4.0",
+    "@typescript-eslint/parser": "^7.4.0",
```

Comment:
Why did you upgrade this lib? It was 4 majors before.

Reply:
Reverted all package changes related to eslint, I'll do it in this PR:
https://github.com/HathorNetwork/hathor-wallet-service/pull/156